### PR TITLE
Revert "Feedback"

### PR DESCRIPTION
**Pull request by sti-x**
Reverts Steamledge-Pythoneers/phase-2-assignment-1-sti-x#1

### CODE REVIEW

**Review by drak00 — CHANGES_REQUESTED** [review]
**Note**
After your unauthorized merge I reverted to a new branch for you (revert-1-master) this is now the branch I'll be grading.

Your new branch (revert-1-master) fails the unit tests. 
ensure they pass all unit tests (Example/ folder and Exercise/ folder)

File: Exercise/compute_lowest_terms.py [hunk 1]
```diff
@@ -1,4 +1,61 @@
...
+
+	# Converting strings to integers
+	cont = [int(i) for i in x]
+
+	# copy is a copy of the prepared input that wouldn't be altered
+	copy = cont[:]
+
+	"""
+	Looping through the minimum value to obtain factors
+	"""
```
Comment (drak00) [inline Exercise/compute_lowest_terms.py]: only use this for docstrings. comments use #

File: Exercise/compute_lowest_terms.py [hunk 2]
```diff
@@ -1,4 +1,61 @@
...
+	factors = [i+1 for i in range(min(cont)) if min(cont) % (i+1) == 0]
+
+	# Reversing the list allows for dividing by larger common factors first before smaller ones
+	factors.reverse()
+
+	"""
+	Division by common factors
+	"""
```
Comment (drak00) [inline Exercise/compute_lowest_terms.py]: not a comment

File: Exercise/compute_lowest_terms.py [hunk 3]
```diff
@@ -1,4 +1,61 @@
...
+	# Converting integers back to strings
+	cont = [str(i) for i in cont]
+
+	# Removing decimal points caused by division process
+	result = [i.split('.') for i in cont]
+
+	"""
+	Using if conditions to handle division of and by zero and also to decide
+	return values
+	"""
```
Comment (drak00) [inline Exercise/compute_lowest_terms.py]: change to comment

**Review by drak00 — CHANGES_REQUESTED** [review]
fails Example/ folder unit test

fails Exercise/ folder unit test

add 'gothca' unit test from slack

**Comment by sti-x** [comment]
I hope it works this time, made another push

**Review by drak00 — APPROVED** [review]
Great! though you didn't add the gotcha test function, It passed when I added it locally.
Also one of your commits was not in imperative voice. I'll let that slide this once, remember to always use 
imperative tone for commit messages in the future.

**Comment by sti-x** [comment]
Noted Boss, thank you for being so patient with me during this learning process.

**Comment by drak00** [comment]
We were all there at some point, never give up and you'll master all these in no time. All the best

**Pull request merged**